### PR TITLE
LPS-129260 Crypto Review

### PR DESCRIPTION
**Pull request by martamedio**

### CODE REVIEW

File: modules/apps/portal-crypto-hash/portal-crypto-hash-provider/portal-crypto-hash-provider-bcrypt/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bcrypt-crypto-hash-provider-configuration-name=BCrypt Crypto Hash Provider Configuration
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash/portal-crypto-hash-provider/portal-crypto-hash-provider-bcrypt/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

File: modules/apps/portal-crypto-hash/portal-crypto-hash-provider/portal-crypto-hash-provider-message-digest/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+message-digest-algorithm=Message Digest Algorithm
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash/portal-crypto-hash-provider/portal-crypto-hash-provider-message-digest/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

File: modules/apps/portal-crypto-hash/portal-crypto-hash-spi/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+crypto-hash-provider-configuration-name=Hash Provider Configuration Name
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash/portal-crypto-hash-spi/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by arthurchan35** [comment]
Merged in upstream.

**Pull request closed without merging**